helm/superset/Chart.yaml

```diff
@@ -22,7 +22,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.7.2
+version: 0.7.3
```

Comment:
Prob need to bump this one more time

Reply:
Did not know which one you would merge first :)